exercise-book/src/nrf52-tools.md

```diff
@@ -66,13 +71,66 @@ To access the USB devices as a non-root user, follow these steps:
     crw-rw-r--+ 1 root root 189, 16 May 20 12:11 /dev/bus/usb/001/017
     ```
 
-    The `+` part in `crw-rw-r--+` indicates the device can be accessed without `root` permissions.
+The `+` part in `crw-rw-r--+` indicates the device can be accessed without `root` permissions.
```

Comment:
```suggestion
The `+` part in `crw-rw-r--+` indicates the device can be accessed without `root` permissions. If you have permission to access them dongle, then the nRF52-DK should also work because both were listed in the udev rules file.
```